### PR TITLE
[IMP] project, project_issue : CC email is set as follower on project…

### PR DESCRIPTION
**Pull request by ara-odoo**
… task and issue
Task: https://www.odoo.com/web#id=13484&view_type=form&model=project.task&menu_id=3942&action=327
Pad: https://pad.odoo.com/p/openerp-project.task-32UME9RS4K

### CODE REVIEW

File: addons/project/project.py [hunk 1]
```diff
@@ -896,13 +896,22 @@ def message_new(self, cr, uid, msg, custom_values=None, context=None):
...
-        email_list = tools.email_split((msg.get('to') or '') + ',' + (msg.get('cc') or ''))
+        email_data = ('%s,%s') % (msg.get('cc'), msg.get('from'))
+        email_list = tools.email_split(email_data)
         new_task = self.browse(cr, uid, res, context=context)
         if new_task.project_id and new_task.project_id.alias_name:  # check left-part is not already an alias
             email_list = filter(lambda x: x.split('@')[0] != new_task.project_id.alias_name, email_list)
-        partner_ids = filter(lambda x: x, self._find_partner_from_emails(cr, uid, email_list, check_followers=False))
+        partner_ids = []
+        for email in email_list:
+            partner_id = filter(lambda x: x, self._find_partner_from_emails(cr, uid, [email], check_followers=False))
```
Comment (hmo-odoo) [inline addons/project/project.py]: can find partner from email out side for loop ?

File: addons/project/project.py [hunk 2]
```diff
@@ -896,13 +896,22 @@ def message_new(self, cr, uid, msg, custom_values=None, context=None):
...
+            if not partner_id:
+                for name_email in email_data.split(','):
```
Comment (hmo-odoo) [inline addons/project/project.py]: may be simple logic:
1) prepare dict like {'email': name_email} from email_data
1) prepare todo email list from email list which does not link with any partner.
2) create partner with full name (name_email) for all todo email list

File: addons/project/project.py (hunk 1 above)
Comment (gan-odoo) [inline addons/project/project.py]: loop was removed.

File: addons/project/project.py (hunk 2 above)
Comment (gan-odoo) [inline addons/project/project.py]: @hmo-odoo I made changes as per your instruction.
Thanks.

File: addons/project/project.py [hunk 3]
```diff
@@ -440,6 +440,15 @@ def write(self, cr, uid, ids, vals, context=None):
             vals.update(alias_model_id=model_ids[0])
         return super(project, self).write(cr, uid, ids, vals, context=context)
 
+    def _find_create_partner_from_mail(self, cr, uid, email_list, name_email, context=None):
```
Comment (hmo-odoo) [inline addons/project/project.py]: can you move this method in res.partner model in base module ?

and also check this method: find_or_create in res.partner. can not use this method ?

**Comment by hmo-odoo** [comment]
merged at https://github.com/odoo/odoo/commit/bb1200adc781efc1318e502a84cc1e77315aabc4

**Pull request closed without merging**